### PR TITLE
Update NL.xml with missing translations

### PR DESCRIPTION
**Pull request by Mivaweb**
### Prerequisites

- [ ] I have written a descriptive pull-request title
- [ ] I have linked this PR to an issue on the tracker at http://issues.umbraco.org

### CODE REVIEW

File: src/Umbraco.Web.UI/umbraco/config/lang/nl.xml [hunk 1]
```diff
@@ -1146,7 +1153,12 @@ Echter, Runway biedt een gemakkelijke basis om je snel op weg te helpen. Als je
     <key alias="thisEditorUpdateSettings">die gebruik maken van deze editor zullen geupdate worden met deze nieuwe instellingen</key>
 
     <key alias="memberCanEdit">Lid kan bewerken</key>
+    <key alias="memberCanEditDescription">Toestaan dat deze eigenschap kan worden gewijzigd door het lid op zijn profiel pagina.</key>
+    <key alias="isSensitiveData">Omvat gevoelige gegevens</key>
+    <key alias="isSensitiveDataDescription">Verberg deze eigenschap voor de content editor dat geen toegang heeft tot het bekijken van gevoelige informatie.</key>
```
Comment (dawoe) [inline src/Umbraco.Web.UI/umbraco/config/lang/nl.xml]: Shouldn't dat be die

Comment (Mivaweb) [reply src/Umbraco.Web.UI/umbraco/config/lang/nl.xml]: @dawoe die geen toegang heeft?

Comment (dawoe) [reply src/Umbraco.Web.UI/umbraco/config/lang/nl.xml]: yep.."Verberg deze eigenschap voor de content editor die geen..."

**Comment by abjerner** [comment]
@Mivaweb Thanks for creating the PR 👍 

@dawoe @nul800sebastiaan this seems like a low hanging fruit to get merged - can one of you confirm the recent changes? I checked with Google Translate - but since you're both Dutch...

**Comment by nul800sebastiaan** [comment]
I made that mistake once.. @dawoe is Belgian 😂😁

**Comment by abjerner** [comment]
@nul800sebastiaan Oh - my bad. Then I'm counting on you ;)

**Comment by dawoe** [comment]
@abjerner I speak dutch as well

**Comment by nul800sebastiaan** [comment]
Thanks for the help @Mivaweb and @dawoe ! 👍

**Pull request merged**